### PR TITLE
Add DbInitialize and fix Mapping

### PR DESCRIPTION
**Pull request by vlad12345123**

### CODE REVIEW

File: exalearn-api/ExaLearn.Bl/Mapping/QuestionProfile.cs [hunk 1]
```diff
@@ -17,6 +17,10 @@ public QuestionProfile()
                 .ForMember(t => t.AuditionQuestion, map => map.MapFrom(source => source))
                 .ForAllOtherMembers(x => x.Ignore());
 
+            CreateMap<TopicQuestionDTO, TestDTO>()
+                .ForMember(t => t.EssayTopic, map => map.MapFrom(source => source.Topic))
+                .ForAllOtherMembers(x => x.Ignore());
+
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Mapping/QuestionProfile.cs:24]: t.Speaking?

File: exalearn-api/ExaLearn.WebApi/appsettings.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=;Database=exalern_db",
+    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=Pass12345;Database=exalern_db",
     "HostingContext": "linkToFilesHosting"
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.WebApi/appsettings.json:12]: remove your local settings for Password field

Comment (vlad12345123) [reply exalearn-api/ExaLearn.WebApi/appsettings.json:12]: this password is required to connect to the server Database. if you need access to the local database, just change it.

File: exalearn-api/ExaLearn.Bl/Services/QuestionService.cs [hunk 3]
```diff
@@ -29,8 +29,7 @@ public async Task<TestDTO> GenerateTestAsync(LevelType level)
             
             return _mapper.Map<TestDTO>(grammarQuestions)
                 .Map(auditionQuestions)
-                .Map(essayTopic)
-                .Map(speakingTopic);       
+                .Map(new { essayTopic, speakingTopic });       
         }
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Services/QuestionService.cs:31]: add test to GenerateTestAsync

File: exalearn-api/ExaLearn.WebApi/appsettings.json (hunk 2 above)
Comment (artemgael96) [reply exalearn-api/ExaLearn.WebApi/appsettings.json:12]: that why you must to create new file appsettings.exalearn.ru.json and must to move settings of server to this file, the expected result will be two files : first file with local setting and second file with server settings

**Pull request closed without merging**